example/src/components/ChatboxTest.tsx

```diff
@@ -27,51 +27,52 @@ import {
 import { hooks, useMutation } from '../config/queryClient';
 import ChatboxWrapper from './ChatboxWrapper';
 
+const TextInputControl = styled(FormControlLabel)({
+  width: '100%',
+});
+
+const Container = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+});
+
+const TestContainer = styled('div')(({ theme }) => ({
+  margin: theme.spacing(1),
```

Comment:
You might test, but I think you can use `m: 1` it will use by default the theme spacing for margin! Same goes for padding.

Reply:
Sadly the shorthand properties do not work in `styled` only in `sx prop` 😢